### ISSUE TITLE
Warden of the Hellborn for warlock is still unmarketable

### ISSUE THREAD

**Issue by notgee** [ValveSoftware/Dota2-Gameplay#6186]
### Cosmetic Item Name

Warden of the Hellborn

### Description

Warden of the Hellborn, head slot immortal for Warlock from Immortal treasure I 2020, is still unavailable for sell or trade, unlike other immortals from treasure.

### Example Match ID

none

### Screenshots

![image](https://user-images.githubusercontent.com/121366312/209454483-da37787e-ab93-48b5-a04e-9b87dc3f4707.png)
![image](https://user-images.githubusercontent.com/121366312/209454615-85474a87-f15d-4202-b3e6-78dcfdeedf0f.png)

**Comment by shaileshgarg** [comment]
#5284

**Comment by notgee** [comment]
> #5284

nope, other ones from mentioned ticket are available already, unlike this exact item.

**Comment by shaileshgarg** [comment]
I have added the list of items unmarketable in the other thread, you can close this one, and upvote that one.
